hawkauthlib/__init__.py

```diff
@@ -71,8 +81,10 @@ def sign_request(request, id_, key, algorithm=None, params=None):
         params["ts"] = str(int(time.time()))
     if "nonce" not in params:
         params["nonce"] = utils.b64encode(os.urandom(5))
+    if include_payload_hash:
+        params["hash"] = hash_payload(request, algorithm)
     # Calculate the signature and add it to the parameters.
-    params["mac"] = get_signature(request, key, algorithm, params, accept_untrusted_content=True)
+    params["mac"] = utils.get_signature(request, key, params, algorithm)
```

Comment:
I've pushed the "calculate signature based on trusted content hash" logic down into a utility function, which I think makes it a bit easier to understand the behaviour of the top-level functions here.

Reply:
`ALGORITHMS` is imported from the `__init__` into `utils` and `__init__` imports `utils`, it's a circular reference and tests will be failing this PR for sure